### PR TITLE
PLAT-584: SMVFindCallRequestAwaitSMCallSummary test

### PR DESCRIPTION
**Pull request by dgovorukhin**
Added  new test for testing logic in SMVFindCallRequest

https://insolar.atlassian.net/browse/PLAT-584

https://insolar.testrail.io/index.php?/cases/view/5435&group_by=cases:section_id&group_id=948&group_order=asc

### CODE REVIEW

File: ledger-core/virtual/handlers/vfindcallrequest_test.go [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/testutils"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/gen"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/messagesender"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/longbits"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/callregistry"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/callsummary"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/testutils/shareddata"
+	"github.com/stretchr/testify/require"
```
Comment (sexyhamster) [inline ledger-core/virtual/handlers/vfindcallrequest_test.go:27]: import order

Comment (dgovorukhin) [reply ledger-core/virtual/handlers/vfindcallrequest_test.go]: Fixed

File: ledger-core/virtual/handlers/vfindcallrequest_test.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/gen"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/predicate"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/descriptor"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/object"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/testutils/slotdebugger"
+)
+
+func TestVFindCallRequestAwaitCallSummarySM(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (bigbes) [reply ledger-core/virtual/handlers/vfindcallrequest_test.go:26]: Also split name into two parts:
```
TestVFindCallRequest_AwaitCallSummarySM
```

[the remark this answers is not in the document]
Comment (dgovorukhin) [reply ledger-core/virtual/handlers/vfindcallrequest_test.go]: Fixed

**Pull request merged**